tests/rt.conf

```diff
@@ -322,9 +322,53 @@ RUN | atmaero_control_p8_rad_micro                      | - noaacloud
 #RUN | regional_atmaq                                    | - jet s4                             | baseline |
 
 COMPILE | atmaq_debug | intel | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2 -DDEBUG=ON -D32BIT=ON | - jet noaacloud s4 | fv3 |
-RUN | regional_atmaq_debug                              | - hera jet s4 noaacloud                   | baseline |
+RUN | regional_atmaq_debug                              | - hera jet s4 noaacloud              | baseline |
+
+### ATM-FBH test ###
+COMPILE | atm_fbh | intel | -DAPP=ATMF -DCCPP_SUITES=FV3_HRRR -D32BIT=ON | - wcoss2 noaacloud acorn | fv3 |
+RUN | cpld_regional_atm_fbh                             | - wcoss2 noaacloud acorn             | baseline |
+
+
+
+#############################
+### Intel (LLVM) Tests ######
+#############################
+
+
+# Coupled gfs17 tests on Gaea
+COMPILE | s2swa_32bit_pdlib       | intelllvm | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON            | + gaea | fv3 |
+RUN | cpld_control_gfsv17                               | + gaea                               | baseline |
+
+COMPILE | s2swa_32bit_pdlib_debug | intelllvm | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON | + gaea | fv3 |
+RUN | cpld_debug_gfsv17                                 | + gaea                               | baseline |
+
```

Comment:
Can we also add cpld_control_sfs and cpld_control_p8?

Reply:
Added. See 7e47213